src/descriptor/mod.rs

```diff
@@ -68,6 +73,126 @@ pub enum Descriptor<Pk: MiniscriptKey> {
     ShWsh(Miniscript<Pk, Segwitv0>),
 }
 
+#[derive(Debug, Eq, PartialEq)]
+pub enum DescriptorPublicKey {
+    PukKey(bitcoin::PublicKey),
```

Comment:
typo? Pubkey instead of PukKey?

Reply:
Definitely is.. Thanks!

Reply:
Corrected for the sake of clean history.